src/modules/common/components/LoadingSpinner.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import styles from '../styles/LoadingSpinner.module.css'
+
+export const LoadingSpinner = () => {
+    return (
+        <div className={styles.container}>
+            <div className={styles.spinner}>
+                <div></div>
+                <div></div>
+                <div></div>
+                <div></div>
+                <div></div>
+                <div></div>
+                <div></div>
+                <div></div>
+                <div></div>
+                <div></div>
+                <div></div>
+                <div></div>
```

Comment:
What were all these divs used for?
Its pretty much never needed adding any div into the DOM if it doesnt have a child.

EDIT: Ok just saw the CSS, look i think its probably better to just use an external package over this approach.
this package seems good
https://www.npmjs.com/package/react-spinners
Clip loader looks good for inline. 
https://www.davidhu.io/react-spinners/

Guess we can just remove this file completely if we use a library then

Reply:
done